.github/workflows/release-container-image.yaml

```diff
@@ -1,14 +1,9 @@
 name: Release Container Image
 
 on:
-  schedule:
-    - cron: '3 3 * * *'
-  pull_request:
   push:
     branches: main
```

Comment:
Is this triggered on every push to main? Does this event occur when using mergin pull requests?

Reply:
Yes